scenarios/waymo/gen_sumo_map.py

```diff
@@ -0,0 +1,168 @@
+import argparse
+import io
+import os
+import subprocess
+import xml.dom.minidom
+import xml.etree.ElementTree as ET
+from pathlib import Path
+from typing import List, Tuple
+
+from smarts.sstudio.genhistories import Waymo
+from waymo_open_dataset.protos import scenario_pb2
+
+os.environ["TF_CPP_MIN_LOG_LEVEL"] = "3"
```

Comment:
Why was the "TF_CPP_MIN_LOG_LEVEL" set to 3? Error messages are likely to be suppresed at log level 3. Perhaps, can use log level 2.

Reply:
Its not needed now. Removed!